neurow/lib/neurow/jwt_auth_plug.ex

```diff
@@ -124,13 +124,20 @@ defmodule Neurow.JwtAuthPlug do
     end)
   end
 
-  defp jwt_token_from_request(conn) do
-    case conn |> get_req_header("authorization") do
-      ["Bearer " <> jwt_token] ->
-        {:ok, jwt_token}
+  @auth_header_keys ["x-interservice-authorization", "authorization"]
```

Comment:
For the public API, we want to continue using the regular `authorization` HTTP header. `x-interservice-authorization` should only be used for backend to backend communications.

I think we should pass the name of the authorization header as a configuration parameter when jwt_auth_plug is added to plug pipeline.

Reply:
oops yes, get it!

Reply:
fixed ✅